### PR TITLE
Easy run with bat file (with requirements check and install if needed)

### PR DESCRIPTION
**Pull request by DenTheProgrammer**
### Background

With that change users now can clone/pull the repo, optionally update .env file and just run the script
### Changes

Added run.bat, run_continuous.bat and check_requirements.py
### Documentation

check_requirements.py checks for missing packages and does packages installation only if it's needed
### Test Plan

Deleted some packages, run the bat file, works every time
### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thouroughly tested my changes with multiple packages configurations
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Review by Slowly-Grokking — DISMISSED** [review]

**Review by SuperYeti — APPROVED** [review]

**Review by nponeccop — APPROVED** [review]

File: run_continious.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@echo off
```
Comment (richbeales) [inline run_continious.bat]: please spell this file correctly.

Comment (DenTheProgrammer) [reply run_continious.bat]: Done

**Review by nponeccop — APPROVED** [review]

**Review by richbeales — APPROVED** [review]

**Comment by itsmarble** [comment]
Thank you for the contribution! This looks like a useful addition to the project. I have a question, but please bear with me as I'm not very experienced with this topic. I was wondering if these changes have been tested with a virtual environment setup, such as using pyenv. Do the check_requirements.py script and the provided .bat files work as expected when the packages are installed within a virtual environment? I apologize if this is a basic question, but I would appreciate any insights you can provide. Thanks!

**Comment by DenTheProgrammer** [comment]
> Thank you for the contribution! This looks like a useful addition to the project. I have a question, but please bear with me as I'm not very experienced with this topic. I was wondering if these changes have been tested with a virtual environment setup, such as using pyenv. Do the check_requirements.py script and the provided .bat files work as expected when the packages are installed within a virtual environment? I apologize if this is a basic question, but I would appreciate any insights you can provide. Thanks!

Teested it now, seems to work fine with pyenv

**Pull request merged**